rcon/steam_utils.py

```diff
@@ -40,6 +45,38 @@ def get_steam_profiles(steam_ids):
         logging.exception('Unexpected error while fetching steam profile')
         return None
 
+@ttl_cache(60 * 5, cache_falsy=False, is_method=False)
+def get_hll_playtime_in_hours(steam_id):
+    if not STEAM_KEY:
+        return None
+    try:
+        api = WebAPI(key=STEAM_KEY)
+        # https://developer.valvesoftware.com/wiki/Steam_Web_API#GetOwnedGames_.28v0001.29
+
+        
+        response = api.IPlayerService.GetOwnedGames_v1(
+            steamid=steam_id, 
```

Comment:
It's not possible to query that in bulk? Just like we do in `get_steam_profiles` 

The `get_steam_profiles` function is used as a cron job once a day to refresh the steam profiles of the players in your DB btw, so if you'd just inject that info in the return value of `get_steam_profiles` itself then you get that for free.
The only down side is that it's a bit less nice when accessing the value when compared to the prime attribute that you added.